src/Schema/CMS/Events/BatchImportFlow.ts

```diff
@@ -0,0 +1,164 @@
+/**
+ * Schemas describing CMS BatchImportFlow events
+ * @packageDocumentation
+ */
+
+import { CmsContextModule } from "../Values/CmsContextModule"
+import { CmsOwnerType } from "../Values/CmsOwnerType"
+import { CmsActionType } from "./index"
+
+/**
+ * A partner has clicked the import button and started the flow.
+ *
+ * @example
+ * ```
+ * {
+ *   context_module: "batchImportFlow",
+ *   context_page_owner_type: "batchImport",
+ *   context_page_owner_id: "67b646ecbe87376bfeb3f962",
```

Comment:
@leodmz - Is this the event for the button on the top of the /artworks listing page? If so, we don't yet have the batch import ID to use here. Thoughts?